client/my-sites/theme/main.jsx

```diff
@@ -760,6 +782,20 @@ class ThemeSheet extends Component {
 		);
 	};
 
+	onThemeActive = () => {
+		if ( ! this.state.isAtomicTransferCompleted ) {
+			this.setState( {
+				isAtomicTransferCompleted: true,
+			} );
+
+			const { isAtomic, siteSlug } = this.props;
+			if ( ! isAtomic ) {
+				const newSiteSlug = siteSlug.replace( /\b.wordpress.com/, '.wpcomstaging.com' );
+				return page( `/theme/makoney/${ newSiteSlug }` );
+			}
```

Comment:
Bug: You've hard-coded `makoney` in the URL.
```suggestion
			const { isAtomic, siteSlug, themeId } = this.props;
			if ( ! isAtomic ) {
				const newSiteSlug = siteSlug.replace( /\b.wordpress.com/, '.wpcomstaging.com' );
				return page( `/theme/${ themeId }/${ newSiteSlug }` );
			}
```